plans/tier1.fmf

```diff
@@ -249,3 +249,20 @@ description+: |
         discover+:
             test+<:
                 - conversion-method/rhsm_conversion
+
+    /host-metering:
+        enabled: false
+        adjust+:
+            # TODO: enable only when running on AWS, GCP, Azure
```

Comment:
@danmyway @kokesak is there anything we need to do here?

Reply:
This might not be needed as the test is setting `    os.environ["CONVERT2RHEL_CONFIGURE_HOST_METERING"] = "force"` and using fake endpoint (localhost).

Reply:
All the images used are in AWS. Only that centos-7 and centos-8 images are our own there (not publicly available in the AWS marketplace) the rest are the official ones. I think we don't need this condition at all.